src/components/atoms/HomeArrow.vue

```diff
@@ -1,13 +1,50 @@
 <template>
-	<div class="home__arrow-outline">
+	<div v-on:click="traverse(arrow, containers)" class="home__arrow-outline">
 		<img v-bind:src="arrow.src" v-bind:alt="arrow.alt" class="home__arrow">
 	</div>
 </template>
 
 <script>
 	export default {
 		name: "homeArrow",
-		props: ["arrow"]
+		props: ["arrow", "containers"],
+		methods: {
+			traverse: function(arrow, containers) {
+				console.log(containers);
+				const length = containers.length;
+				
+				let active = 0;
+				for (; active < length-1 ; active++) {
+					if (!containers[active].isHidden){
+						break;
+					}
+				}
+				if (arrow.direction == 'left'){
+					containers[active].isTop = 1;
```

Comment:
why not `true` and `false` why 1 and 0 
If its numeric I expect to pass to this property all sorts of numbers like 1126 1326512 -421